### ISSUE TITLE
Proposed changes to membrane terms

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#23029]
Hello, 

While discussing GO-CAM modeling guidelines with @ukemi @vanaukenk @thomaspd , we decided that we should go ahead and make some changes in the membrane terms, which have been problematic for a long time and discussed multiple times. Here's what we propose to do: 

New structure: 
* Plasma membrane
* * Spanning
* * Anchored (tail or lipid-anchored)
* * * Anchored on cytoplasmic side
* * * Anchored on external side
* Extracellular space
* * Extracellular space adjacent to the plasma membrane (rename GO:0031232 extrinsic component of external side of plasma membrane)
* Cytosol
* * cytosol adjacent to the plasma membrane (GO:0031234 extrinsic component of cytoplasmic side of plasma membrane)

Actions: 

- [ ] Merge ‘intrinsic to membrane’ with membrane
- [ ] Merge spanning component of membrane into integral component of membrane
- [ ] Move ‘extrinsic’ components out of ‘membrane’ + change labels from ‘extrinsic’ to ‘adjacent’

To be discussed at the next annotation call to get feedback from curators.

Thanks, Pascale

**Comment by ValWood** [comment]
I like this.

When you say " Merge spanning component of membrane into integral component of membrane"
do you propose to call this term "spanning"? (i.e. above). Spanning seems more accurate because integral by the current definition covers anchored.
"having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane.
"

**Comment by pgaudet** [comment]
Right, I would prefer the term labels be as proposed in the structure here: https://github.com/geneontology/go-ontology/issues/23029#issue-1171288260

There is a lot of confusion between intrinsic and integral, that would solve that by removing both as term labels.

**Comment by ValWood** [comment]
> Right, I would prefer the term labels be as proposed in the structure here: https://github.com/geneontology/go-ontology/issues/23029#issue-1171288260

I agree. I think I misunderstood this:

"Merge spanning component of membrane into integral component of membrane"

(I agree that spanning is easier to understand than integral)
